packages/angular/common/src/directives/navigation/tabs.ts

```diff
@@ -39,8 +41,29 @@ export abstract class IonTabs implements AfterContentInit, AfterContentChecked {
 
   private tabBarSlot = 'bottom';
 
+  private hasTab = false;
+  private selectedTab?: any;
```

Comment:
Using `any` here is overriding the type signature of treating it as optionally defined (`undefined`). This means you are missing out on potential errors on L203, L206 and L212.

I would recommend minimally updating the type signature to:
```ts
private selectedTab?: { tab: string };
```

and then handling those type errors that arise for the `selectedTab` not being defined.

Reply:
https://github.com/ionic-team/ionic-framework/pull/29786/commits/99d70171fb1a626954832fe19c6f36e398d5e195